spec/hangman_rules_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+require_relative '../lib/hangman_rules'
+
+RSpec.describe HangmanRules do
```

Comment:
I think this is the key set of tests/facts about the game.
I would suggest you revisit these and make sure these read like how you'd explain the rules to me while
having a coffee/tea without a laptop.